NuGet.Config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <config>
+    <add key="globalPackagesFolder" value="./src/packages" />
```

Comment:
we have similar settings in SDK with 2 settings, based on building and testing only this one is needed.